bench/Main.hs

```diff
@@ -218,14 +218,16 @@ runBenchmarks benchmarks = do
     putStrLn $ intercalate ", " $ map (showDuration . snd) results
 
 runBench :: HasConfig => Bench -> IO Seconds
-runBench Bench {..} = handleAny (\e -> print e >> return (-1))
+runBench Bench {..} = handleAny (\e -> print e >> return (-2))
```

Comment:
Having -1 as a special case was odd, but just on the edge of tolerable. Now we're having two magic negative seconds I think an enum would be better.